sql_dump.sql

```diff
@@ -46,7 +54,7 @@ INSERT INTO authors (`name`, email) VALUE (
         
     SELECT * FROM patterns;
     
-     INSERT INTO tags (`name`) VALUES 
+INSERT INTO tags (`name`) VALUES 
 		("Beginner"),
         ("Intermediate"),
         ("Advanced"),
```

Comment:
```suggestion
	("Beginner"),
        ("Intermediate"),
        ("Advanced"),
```
Spacing seems a little wonky

Reply:
Yeah it's an SQL dump